src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/DelayLoadHelperMethodImport.cs

```diff
@@ -70,5 +70,20 @@ public override IEnumerable<DependencyListEntry> GetStaticDependencies(NodeFacto
         public override int ClassCode => 192837465;
 
         public MethodDesc Method => _method.Method;
+
+        public override int CompareToImpl(ISortableNode other, CompilerComparer comparer)
+        {
+            DelayLoadHelperMethodImport otherNode = (DelayLoadHelperMethodImport)other;
+            int result = _useInstantiatingStub.CompareTo(otherNode._useInstantiatingStub);
+            if (result != 0) return result;
+
+            result = _signatureContext.CompareTo(otherNode._signatureContext, comparer);
+            if (result != 0) return result;
+
+            result = _method.CompareTo(otherNode._method, comparer);
+            if (result != 0) return result;
+
+            return base.CompareToImpl(other, comparer);
```

Comment:
I see some missing fields in the comparison here: `_helper` and `_delayLoadHelper`

Reply:
Not sure why they are redeclared here... I think a better way to do this is the remove these fields from this derived class if they are not used here

Reply:
Removed.